src/server/users/_test/login.test.ts

```
@@ -73,18 +72,7 @@ describe(`login`, () => {
       []
     );
     const token = await login(`someone@user.com`, user.password);
-    expect(token).not.toBeNull();
-    expect(queryFn).toHaveBeenCalledTimes(2);
-    expect(queryFn).nthCalledWith(
-      1,
-      `SELECT * FROM users WHERE email = $1 RETURNING email, id, digest`,
-      [user.email]
-    );
-    expect(queryFn).nthCalledWith(
-      2,
-      `INSERT INTO sessions (hashed_access_token, expires_at, user_id) VALUES ($1, $2, $3) RETURNING user_id, expires_at;`,
-      [expect.anything(), expect.anything(), user.id]
-    );
+    expect(token).toThrow('email not found!');
```

Changed `login()` to throw an error if email isn't a match instead of just returning a string, ended this test as `expect.toThrow()`. Did the same for the incorrect password test.